src/components/tooltip/markers/MarkerTooltipContents.js

```diff
@@ -151,7 +124,11 @@ function _makePriorityHumanReadable(
   }
 
   prioLabel = prioLabel + '(' + priority + ')';
-  return _markerDetail(key, label, prioLabel);
+  return (
+    <TooltipDetail key={label} label={label}>
```

Comment:
Can we have duplicate labels?

Reply:
In the next commits, we use the key only when needed -- in the current master we were specifying key in a lot of places where I think it wasn't useful (or maybe even detrimental) for React.